custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -232,3 +232,36 @@ outer_predict <- function(input_filename, target_type, model=NULL, positive_clas
     }
     predictions
 }
+
+predict_unstructured <- function(model=NULL, data, ...) {
+    .validate_unstructured_predictions <- function(to_validate) {
+        .is_instance_character_raw_null <- function(v) {
+            is.character(v) || is.raw(v) || is.null(v)
+        }
+
+        .is_instance_list_none <- function(v) {
+            is.list(v) || is.null(v)
+        }
+
+
+        if (.is_instance_character_raw_null(to_validate)) {
+
+        } else if (is.list(to_validate)) {
+            if (length(to_validate) != 2){
+                stop(sprintf("In unstructured mode predictions of type list must have length = 2, got %s", length(to_validate)))
+            }
+            par1 <- to_validate[[1]]
+            par2 <- to_validate[[2]]
+            if (!.is_instance_character_raw_null(par1) || !.is_instance_list_none(par2)) {
```

Comment:
can you even have a list where one of the items is null?

Reply:
```
> list("a", NULL)
[[1]]
[1] "a"

[[2]]
NULL
```

Reply:
***Fixed as you suggested***